docs/root/api-v1/cluster_manager/sds.rst

```diff
@@ -3,7 +3,7 @@
 Service discovery service
 =========================
 
-Service discovery service :ref:`architecture overview <arch_overview_service_discovery_types_sds>`.
+Service discovery service architecture overview.
```

Comment:
Since we still have the v1 docs and haven't deleted, can we just add a new anchor here for v1_sds and then continue to ref link from the previous file that you removed the link from?

Reply:
Done.